src/backend/executor/cypher_utils.c

```diff
@@ -248,6 +248,8 @@ HeapTuple insert_entity_tuple_cid(ResultRelInfo *resultRelInfo,
     }
 
     // Insert the tuple normally
+    // TODO
```

Comment:
Is the TODO completed? Should we remove this comment?

Reply:
Thanks for catching that! I added it to check whether the following line was needed or not.